src/PolicyInsights/PolicyInsights.Management.Sdk/Customizations/Attestation.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
```

Comment:
As an open souce project, we recommend to add the license as header to each files. Here is an [example](https://github.com/Azure/azure-powershell/blob/ef5508408f96ad7212f56813e6cc33ea0f40e5e1/src/PolicyInsights/PolicyInsights.Test/ScenarioTests/AttestationTests.cs#L1-L13)